docs/_ug/commandSummary/itemCommands/find.md

```diff
@@ -1,2 +1,3 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>find n/ITEM_NAME</b><br><br><u>Valid Example:</u><br>find n/Potato
+find ITEM_NAME [ITEM_NAME]
+Note: There can be additional item names provided.
```

Comment:
This is unclear. What does this mean?

Reply:
Ah I meant additional [item_name] parameters, I'll fix this